src/main/java/com/openshift/restclient/model/volume/property/IPersistentVolumeProperties.java

```diff
@@ -0,0 +1,10 @@
+package com.openshift.restclient.model.volume.property;
+
+
+import org.jboss.dmr.ModelNode;
+
+public interface IPersistentVolumeProperties {
+
+	void setProperties(ModelNode node);
```

Comment:
Exposing implementation detail in public interface - workaround for the fact, that the format of type properties can generally differ.
This way I can avoid deciding what type of properties the client code sends, otherwise there would have to be either instanceof or isAssignableFrom to check type of the object it received.